src/test/kotlin/no/nav/melosysskattehendelser/melosys/KafkaTestProducer.kt

```diff
@@ -0,0 +1,29 @@
+package no.nav.melosysskattehendelser.melosys
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import no.nav.melosysskattehendelser.melosys.consumer.HendelseMelding
+import no.nav.melosysskattehendelser.melosys.consumer.VedtakHendelseMelding
+import org.apache.kafka.common.serialization.StringSerializer
+import org.springframework.boot.autoconfigure.kafka.KafkaProperties
+import org.springframework.boot.test.context.TestConfiguration
+import org.springframework.context.annotation.Bean
+import org.springframework.kafka.core.DefaultKafkaProducerFactory
+import org.springframework.kafka.core.KafkaTemplate
+import org.springframework.kafka.core.ProducerFactory
+import org.springframework.kafka.support.serializer.JsonSerializer
+
+@TestConfiguration
+class KafkaTestProducer {
+
+    @Bean
+    fun testKafkaTemplate(
+        kafkaProperties: KafkaProperties,
+        objectMapper: ObjectMapper?
+    ): KafkaTemplate<String, VedtakHendelseMelding> {
+        val props = kafkaProperties.buildProducerProperties()
```

Comment:
Denne kommer opp som `Deprecated since 3.2.0` så bør ta med null som argument for å unngå denne